### PR TITLE
added ping feature and licences

### PR DESCRIPTION
**Pull request by CookieDecide**

### CODE REVIEW

File: src/apps/fun.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+"""Copyright by CookieDecide, Darkuuu
+Licensed under MIT License
+"""
+
```
Comment (Darkuuu) [inline src/apps/fun.py:4]: why new line?

**Review by Darkuuu — COMMENTED** [review]
Please take a look at my comment in fun.py and explain.

File: src/apps/fun.py (hunk 1 above)
Comment (CookieDecide) [reply src/apps/fun.py:4]: Because it looks better.

Comment (Darkuuu) [reply src/apps/fun.py:4]: Ah, makes perfect sense. LGTM, merge approved

**Pull request merged**